transform.js

```diff
@@ -27,7 +27,7 @@ const external = () => {
     name: 'rollup-plugin-external',
     resolveId(id) {
       // filter relative or absolute imported modules
-      if (builtins.includes(id) || !id.startsWith('.') || !isAbsolute(id)) {
+      if (builtins.includes(id) || (!id.startsWith('.') && !isAbsolute(id))) {
```

Comment:
Relative file is external, and it's used to suppress the console warning:

```
➜  rollup-jest git:(pr/13) ✗ yarn jest --no-cache
yarn run v1.22.10
'..' is imported by test/transform.spec.js, but could not be resolved – treating it as an external dependency
 PASS  test/main.spec.js
 PASS  test/transform.spec.js
```

Reply:
Your code here suggested that you wanted to exclude relative files. In reality, we don't want to import anything, so we should just return false here. Then we could combine that with the first plugin.